skimage/transform/_warps.py

```diff
@@ -6,6 +6,14 @@
                          _convert_warp_input, _clip_warp_output)
 
 
+def _multichannel_default(multichannel, ndim):
+    # utility for maintaining previous color image default behavior
+    if ndim == 3:
+        return True
+    else:
+        return False
```

Comment:
@stefanv:  One other related question: If I add a deprication warning as above, then several existing tests fail due to the warning being raised.  Should I modify these tests to explicitly set the `multichannel` argument?  (or is there a way to tell the tests to ignore that warning so we can make sure the default ``multichannel=None`` result matches prior versions?)